assets/icons/git.svg

```diff
@@ -0,0 +1 @@
+<svg fill="#000000" width="28" height="28" viewBox="0 0 512 512" id="Flat" xmlns="http://www.w3.org/2000/svg"><path d="M456 135.996a80 80 0 1 0 -104.39 76.184 24.02 24.02 0 0 1 -23.608 19.816L184 232a71.6 71.6 0 0 0 -24 4.112v-23.8a80 80 0 1 0 -48 0v87.37a80.014 80.014 0 1 0 48.39 0.132A24.02 24.02 0 0 1 184 280l144.002 -0.004a72.08 72.08 0 0 0 71.854 -67.634A80.14 80.14 0 0 0 456 135.996m-320 -32a32 32 0 1 1 -32 32 32.04 32.04 0 0 1 32 -32M136 408a32 32 0 1 1 32 -32 32.04 32.04 0 0 1 -32 32M376 167.996a32 32 0 1 1 32 -32 32.04 32.04 0 0 1 -32 32"/></svg>
```

Comment:
We already have a Git icon here: https://github.com/zed-industries/zed/blob/main/assets/icons/file_icons/git.svg

Can we use the source of that one instead of a brand new icon?

Reply:
Done